src/text.ts

```diff
@@ -193,18 +229,20 @@ export const login = async ({
     // if (yamlConfig.carrierRegexFilter)  {
     //
     // }
-    //
-    // only fetch info once
-    if (user.twilio.countryCode === undefined || user.twilio.countryCode === null) {
-      try {
-        const result = await getCarrier(phone)
-        user.twilio = result
-        await user.save()
-      } catch (err) {
-        // Carrier fetching is a non-critical operation
-        // Primarily useful for analytics
-        // Hence failure should be handled with a warn instead of an error
-        subLogger.warn({ err }, "impossible to fetch carrier")
+
+    if (yamlConfig.sms_provider.toLowerCase() === "twilio") {
```

Comment:
@jrbuhl93 for now I actually think we still want to call the `getCarrier` API. you can remove this `if` here (it's giving metadata on the phone number, specifically we want to check whether it's a voip number, because we don't give sats for the quiz from a voip number)